users/src/test/java/io/crate/operation/auth/AuthenticationServiceTest.java

```diff
@@ -31,48 +34,52 @@
 import static io.crate.operation.auth.AuthenticationService.Matchers.isValidAddress;
 import static io.crate.operation.auth.AuthenticationService.Matchers.isValidUser;
 import static org.hamcrest.core.Is.is;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 
 public class AuthenticationServiceTest extends CrateUnitTest {
 
-    private static final ImmutableMap<String, String> HBA_1 = ImmutableMap.<String, String>builder()
```

Comment:
please leave the tests as they are. additionally you can write a test that converts `Settings` into `Map<String, Map<String, String>`